src/pybind/mgr/volumes/module.py

```diff
@@ -49,7 +49,8 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
         {
             'cmd': 'fs volume create '
                    f'name=name,type=CephString,goodchars={goodchars} '
-                   'name=placement,type=CephString,req=false ',
+                   'name=placement,type=CephString,req=false '
+                   'name=max_mds,type=CephInt,req=false',
```

Comment:
I'm not sure why this needs to be part of the volume creation API. Why not just use a separate RPC to update `max_mds` to what is in the spec?